ui/src/pages/Recovery/Recover.js

```diff
@@ -0,0 +1,227 @@
+import { useEffect, useState } from "react"
+import { appWindow, LogicalSize } from '@tauri-apps/api/window';
+import { Outlet } from "react-router-dom";
+import { useNavigate, useLocation } from "react-router-dom";
+const bip39 = require('bip39');
```

Comment:
I'm sure there's some reason this needs to be `required`, right?

Reply:
The library docs use require() instead of import, so I just copied that and it worked, but I also tried to import and kept getting weird errors. https://www.npmjs.com/package/bip39